apps/Gruntfile.js

```diff
@@ -685,10 +703,15 @@ module.exports = function (grunt) {
   grunt.registerTask('dev', [
     'prebuild',
     'newer:sass',
-    'concurrent:watch',
+    'webpack-dev-server:watch',
+//    'concurrent:watch',
     'postbuild',
   ]);
 
+  grunt.registerTask('dev-server', [
+    'webpack-dev-server:watch',
+  ]);
```

Comment:
probably want to ditch this task in favor of just using dev. Or would it be better for people to choose between running webpack-dev-server vs regular webpack?